Public/script.js

```diff
@@ -1,8 +1,37 @@
 let searchBtn = document.getElementById("search-btn");
 let countryInp = document.getElementById("country-name");
 
+function getRandomCountry() {
```

Comment:
I added a new function called getRandomCountry. Within that I created an array of loads of countries (We can add or remove whichever ones we want so that we control the countries that are actually selected) and then used math random to get one.